tailwind.config.js

```diff
@@ -6,7 +6,7 @@ export default {
   ],
   darkMode: "class",
   theme: {
-    extend: {},
+    extend: {   /* *** Error was here with the Bracket for extend **** */
```

Comment:
This was the error that I found that was preventing Tailwind from working properly,  the bracket that needed to wrap around the custom colors.